### ISSUE TITLE
Auras are applied by hidden actors

### ISSUE THREAD

**Issue by jonathanmclaus** [foundryvtt/pf2e#10842]
The conditions from an aura are applied even when the actor with the aura is hidden. This gives away metagame information for nearby / preplaced creatures and I was told this was not intended.

![image](https://github.com/foundryvtt/pf2e/assets/5731591/e170734e-cf1f-47bf-82ab-b941e83ae2e9)

In this example, the Noxious Aura condition is applied to all actors within range, even though the selected actor is hidden.

Rule Element:

```json
{
  "effects": [
    {
      "affects": "all",
      "events": [
        "enter"
      ],
      "uuid": "Compendium.pf2e.bestiary-effects.Item.AL14FXSxpKyyfnOn"
    }
  ],
  "key": "Aura",
  "radius": 30,
  "slug": "noxious-smog",
  "traits": [
    "conjuration",
    "poison"
  ]
}
```

**Comment by TMunsie** [comment]
Does the NPC have the stated ability to suppress their aura at will?

**Comment by TMunsie** [comment]
Also, I've checked the Smog Wraith and its aura is correctly blocked by walls and doors. The mystery man icon makes it not visible. It doesn't make it not present. Are you thinking that it should make it so it acts like the token is not actually present?

**Comment by jonathanmclaus** [comment]
Right. Perhaps my (and others') understanding is wrong. When I asked in the Discord if it was possible to a hidden / invisible actor (invisible as in the mystery man icon, not the PF2E condition), I was told that should be the default behavior. If it is not the default behavior / possible, then this would be a feature request rather than a bug.

**Comment by simonward** [comment]
Duplicate of https://github.com/foundryvtt/pf2e/issues/6577
